src/css/TableView.css

```diff
@@ -0,0 +1,60 @@
+.table-view {
+  overflow: scroll;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 50px;
```

Comment:
This should be `padding-top: 50px` instead of `margin-top`. We want the scrollable area to include the area underneath `Header`.